_translations/pt_BR.yml

```diff
@@ -181,6 +181,7 @@ pt_BR:
     walletelectrum: "A principal característica do Electrum é a rapidez e simplicidade, com baixo uso de recursos. Ele utiliza servidores remotos, responsáveis pela parte mais complicada do funcionamento do Bitcoin, e permite recuperar sua carteira a partir de sua senha pessoal."
     walletmsigna: "mSIGNA é uma carteira avançada mas fácil de usar, com rapidez e simplicidade, escalabilidade de nível empresarial e uma forte segurança. Ela suporta BIP32, transações multi-assinatura, armazenamento off-line, sincronização multidispositivo e backups criptografados eletrônicos e em papel."
     walletbitcoinwallet: "Bitcoin Wallet é fácil de usar e confiável, sendo também seguro e rápido. Sua visão é descentralização e confiança zero; nenhum serviço central é necessário para operações relacionadas com Bitcoin. O aplicativo é uma boa escolha para pessoas não técnicas."
+    walletedgewallet: "Edge é uma carteira móvel de Bitcoin compatível com Segwit, que aumenta o nível de privacidade, segurança e descentralização muito familiares e utilizado pelas pessoas. As carteiras Edge são criptografadas sempre  automaticamente, armazenadas em backup e funcionam até mesmo quando os servidores Edge ficam inativos."
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`